dockerfiles/Dockerfile.openvino

```diff
@@ -84,7 +84,7 @@ RUN apt update && apt -y install --no-install-recommends apt-transport-https ca-
     git clone --recursive -b ${ONNXRUNTIME_BRANCH} ${ONNXRUNTIME_REPO} && \
     /bin/sh onnxruntime/dockerfiles/scripts/install_common_deps.sh && \
     cd onnxruntime/cmake/external/onnx && python3 setup.py install && \
-    cd ${MY_ROOT}/onnxruntime && ./build.sh --config Release --update --build --parallel --use_openvino ${DEVICE} --build_wheel && \
+    cd ${MY_ROOT}/onnxruntime && ./build.sh --config Release --update --build --parallel --use_openvino ${DEVICE} --build_shared_lib --build_wheel && \
```

Comment:
Also add the same --build_shared_lib flag inside the csharp docker file as well.
https://github.com/microsoft/onnxruntime/blob/master/dockerfiles/Dockerfile.openvino-csharp